src/__test__/videoCard.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import VideoCard from '../components/VideoCard';
+import { mockVideos } from '../mockData';
+
+test('render videoCard component', () => {
+  const firstElement = mockVideos?.items[1];
+  render(<VideoCard data={firstElement} />);
+
+  const title = screen.getByAltText(/Video Tour/i);
+  expect(title).toBeInTheDocument();
+});
```

Comment:
Your tests are enough for what you have up to this point, in the next deliverable you may need to include more cases for the new functionalities